docs/api-docs/channels/channels-quick-start.md

```diff
@@ -54,9 +60,10 @@ Accept: application/json
 <div class="CalloutBlock--info">
 <div class="HubBlock-content">
 
-> ### Note
-> * For a list of accepted `type` and `platform` values, see [Channels API Reference](https://developer.bigcommerce.com/api-reference/cart-checkout/channels-listings-api#platform).
-> * For instructions on finding your app's ID, see [Find and App's ID](https://developer.bigcommerce.com/api-docs/apps/tutorials/id).
+> **Note**

```

Comment:
```suggestion
> ### Note

```

Reply:
I changed these from an H3 header to strong text for accessibility reasons. Since "Note" is too vague to add any value to the page's organizational structure and it seems that the goal is to make the text pop visually, emphasizing the text accomplishes that without throwing off screenreaders.